### ISSUE TITLE
After some minutes the video disappears

### ISSUE THREAD

**Issue by DanielWeeber** [shbatm/MMM-RTSPStream#12]
(using wip)
Hey,

after some minutes of successfully displaying the video it just disappears.
It will only come back if I restart pm2/electron.

The log does not show anything when it disappears. Any idea?

Greets

**Comment by shbatm** [comment]
It sounds like OMXPlayer is failing for some reason.  I've found OMXPlayer has terrible error reporting.  You may try just running OMXPlayer from the command line and seeing if it outputs any errors there:
```shell
omxplayer --win "{left}, {top}, {right}, {bottom}" --live --video_queue 4 --fps 30 {url}
```
You can get the boundaries by running the module and checking the MM server log (should be something like "Starting stream1 with args: ...".  Then disable the module and try running it from the console.  Let me know if it does the same thing.

Thanks,
shbatm

**Comment by shbatm** [comment]
This should now be fixed after 442e471934cbb29bdaf19edd0b42fdc77727c9fb and using PM2 to run OMXPlayer with auto-restart. Please re-open if necessary.

**Comment by DanielWeeber** [comment]
If I am being honest: No change. Still massive problems with the stream.
Mostly when restarted it runs fine for a few minutes, then just disappears. Most of the time it will come back after a certain period of time and can either stay for hours or disappear again.

Branch master now.


{
    module: "MMM-RTSPStream",
    position: "bottom_left",
    disabled: true,
    header: "Webcam",
        config: {
        autoStart: true,
        rotateStreams: false,
        rotateStreamTimeout: 10,
        moduleWidth: 354,
        moduleHeight: 240,
        localPlayer: 'omxplayer',
        stream1: {
            name: 'Haustür',
            url: 'rtsp://user:pw@192.168.0.7:554/live/ch2',
            absPosition:{
		 		top: 1360,
		 		right: 450,
		 		bottom: 1600,
		 		left: 76,
		 		},
             },
        }
}
